tcparse/parse.py

```diff
@@ -541,7 +541,8 @@ def post_init(self):
         self.axes = getattr(self, 'Axis', [])
         if not self.axes:
             # Or they can be stored in a separate file, 'NC.xti':
-            self.axes = [item.Axis[0] for item in getattr(self, 'TcSmItem', [])]
+            self.axes = [item.Axis[0] for item in getattr(self, 'TcSmItem', [])
```

Comment:
Makes me think that we should be using this kind of thing instead of pure `flake8` as suggested on Nikea

https://github.com/PyCQA/flake8-bugbear#opinionated-warnings

Reply:
Yeah, I like that idea. This "failure" is very annoying.